Assets/HoloToolkit/UI/Shaders/3DTextShader.shader

```diff
@@ -0,0 +1,18 @@
+﻿Shader "Custom/3DTextShader"{
```

Comment:
So we are consistent with other HoloToolkit shaders, please make this:

Shader "HoloToolkit/3DTextShader"